docs/docs/installation/event-logging.mdx

```diff
@@ -56,5 +56,13 @@ from superset.stats_logger import StatsdStatsLogger
 STATS_LOGGER = StatsdStatsLogger(host='localhost', port=8125, prefix='superset')
 ```
 
+You can also pass metric names that should be ignored via an optional `metric_denylist` parameter:
+
+```
+STATS_LOGGER = StatsdStatsLogger(host='localhost', port=8125, prefix='superset', metric_denylist={"reports.scheduler"})
```

Comment:
I wonder if it's better to promote `metrics_denylist` to be a config that can be overridden.

Reply:
something like 

```suggestion
STATS_LOGGER_METRICS_DENYLIST:  Set[str] = {}
STATS_LOGGER = StatsdStatsLogger(host='localhost', port=8125, prefix='superset', metrics_denylist=STATS_LOGGER_METRICS_DENYLIST)
```

Reply:
Wouldn't that be the same as just overwriting the STATS_LOGGER with different arguments?

Reply:
@zephyring or @craig-rueda mentioned a good point that there's no guarantee that a metric name might not change. What do you all think about having an enum of strings that we can reference instead? It will have to be maintained and updated of course, but at least then we can export it and refer to the enum in the config rather than the string.

Reply:
If promoting to be a config, we can just override the config and use the default `StatsdStatsLogger` logger.
If later on there's any default in the denylist that will be easier to reference and merge/override also